packages/odie-client/src/constants.tsx

```diff
@@ -40,21 +40,111 @@ export const getOdieForwardToZendeskMessage = (): string =>
 		__i18n_text_domain__
 	);
 
-export const getOdieTransferMessage = (): Message[] => {
+export function getFlowFromBotSlug( botSlug: OdieAllBotSlugs ): string {
+	if ( botSlug === 'ciab-workflow-support_chat' ) {
+		return 'commerce-garden';
+	}
+	return 'wpcom';
+}
+
+export const getOdieTransferMessage = ( botSlug: OdieAllBotSlugs ): Message[] => {
 	const isTestMode = isTestModeEnvironment();
-	const content = isTestMode
-		? __( 'No problem. Help is on the way! (staging)', __i18n_text_domain__ )
-		: __( 'No problem. Help is on the way!', __i18n_text_domain__ );
+	const flow = getFlowFromBotSlug( botSlug );
+
+	// Commerce garden has a simplified, single-message flow
+	if ( flow === 'commerce-garden' ) {
+		return [
+			{
+				content: isTestMode
+					? __(
+							"(STAGING VERSION OF ZENDESK) Yes, of course! A Happiness Engineer is jumping in to help you now. They can see your chat with our assistant, so feel free to share any extra details; we'll take it from there.",
+							__i18n_text_domain__
+					  )
+					: __(
+							"Yes, of course! A Happiness Engineer is jumping in to help you now. They can see your chat with our assistant, so feel free to share any extra details; we'll take it from there.",
+							__i18n_text_domain__
+					  ),
```

Comment:
[Nitpick, non-blocking] This is normally a good i18n practice, but given that the test mode does not affect real users, I think we can do some string concatenation here to simplify the code:

```suggestion
				content:
					( isTestMode ? '(STAGING VERSION OF ZENDESK) ' : '' ) +
					__(
						"Yes, of course! A Happiness Engineer is jumping in to help you now. They can see your chat with our assistant, so feel free to share any extra details; we'll take it from there.",
						__i18n_text_domain__
					),
```